### ISSUE TITLE
About EM segmentation

### ISSUE THREAD

**Issue by SJTUMisaka** [tiangexiang/BiO-Net#12]
Thank you for your wonderful work! In your paper you did experiments of EM segmentation. Would you please tell me how to get the data and reproduce your result?

**Comment by tiangexiang** [comment]
Hi, thanks for your interest! 
**Data** we have uploaded the data on our project page for download: https://bionets.github.io/Piriform_data.zip. Please unzip the data and write a simple dataloader to load the tif images. Note that we experimented on EM segmentation as binary segmentation, so you need to process the GT labels to binary first.
**Training** the training configures and network structures are consistent with the ones used for nuclei segmentations. However, you may want to tune the batchsize and learning rate a bit to obtain the best results.
**Evaluation** we used the official evaluation script (with FIJI) to obtain the results, please refer to the official webpage: http://brainiac2.mit.edu/isbi_challenge/evaluation.

**Comment by SJTUMisaka** [comment]
Thank you for your reply!
